app/models/smart_proxy.rb

```diff
@@ -90,6 +90,25 @@ def has_feature?(feature)
     self.features.any? { |proxy_feature| proxy_feature.name == feature }
   end
 
+  def version
+    begin
+      version = { :status => N_('OK'), :message => ProxyAPI::Version.new(:url => url).version }
```

Comment:
:status should be translated?  It would be preferable to put this into rabl if possible.  Have a look too at the LDAP test response and make them the same: app/views/api/v2/auth_source_ldaps/test.json.rabl.